tests/functional/pageobjects/BasePage.js

```diff
@@ -8,17 +8,17 @@ module.exports = class BasePage {
   }
 
 
-  async wait(time = 200) {
+  wait(time = 200) {
```

Comment:
yeah apparently latest version of webdriverio handles all the functions in a sync way.